import pandas as pd
import mysql.connector
from mysql.connector import Error